src/hyperfine/app.rs

```diff
@@ -226,6 +226,16 @@ fn build_app() -> App<'static, 'static> {
                      when trying to benchmark output speed.",
                 ),
         )
+        .arg(
+            Arg::with_name("command-name")
+                .long("name")
```

Comment:
I would prefer `--command-name` for the long option. If we have a short option anyway, the long option should have a self-documenting character IMO... e.g. for shell scripts. If we change it, the man page also has to be updated.

Reply:
sounds good, see commit https://github.com/sharkdp/hyperfine/pull/330/commits/cc6778884f1defba9fbccbef2fd8c5c0dbeaf526